System-Testing-Deployments-and-Upgrades/User-Authorization/index.rst

```diff
@@ -0,0 +1,162 @@
+
+.. figure:: ./_static/love-authorize.png
+    :name: fig-love-auth-list-summary
+
+    LOVE authorization list summary view showing users authorized to command several CSCs.
+    In general the list will be empty, which means no user is authorized to command any CSC, but CSCs can still command each other.
+
+Individual CSCs
+---------------
+
+If you are not using the high-level control classes you can still request authorization, but you will have to handle most of the boiler plate required to format the request.
+For example, the control classes have information about the components it interacts with and, therefore, can automatically produce a list of the CSCs that need requests.
+They can also determine the correct user identity without the user having to know where to get it from.
+
+The request is done via the ``Authorize`` CSC, which parses the request, handle communication with ``LOVE`` to authorize with an administrator and sets the authorizations in the CSCs.
+
+The following is an example of how to request authorization to command the ``MTRotator`` CSC.
+
+.. code-block:: python
+
+  from lsst.ts import salobj
+
+  domain = salobj.Domain()
+
+  # As mentioned above, we need to request the authorization to the Authorize CSC
+  # so we must construct a remote to send the command to it.
+  authorize = salobj.Remote(domain, "Authorize")
+
+  await authorize.start_task
+
+  identity = domain.user_host
+  timeout = 60.0
+
+  # Request authorization to command MTRotator
+  await authorize.cmd_requestAuthorization.set_start(
+    cscsToChange="MTRotator",
+    authorizedUsers=f"+{identity}",
+    timeout=timeout
+  )
+
+  # User is now authorized to command the MTRotator CSC
+  ...
+
+  # Release authorization to command MTRotator
+  await authorize.cmd_requestAuthorization.set_start(
+    cscsToChange="MTRotator",
+    authorizedUsers=f"-{identity}",
+    timeout=timeout
+  )
+
+.. _User-Authorization-Troubleshooting:
+
+Troubleshooting
+===============
+
+Failure to Set Authorization
+----------------------------
+
+One common problem users may face in requesting authorization failure to set authorization by the ``Authorize`` CSC.
+The error message will look like the following:
+  
+  AckError: msg='Command failed', ackcmd=(ackcmd private_seqNum=1003719146, ack=<SalRetCode.CMD_FAILED: -302>, error=1, result="Failed: Failed to set authList for one or more CSCs: **{'ATPtg': 'Timed out waiting for command acknowledgement'}**. The following CSCs were successfully updated: {'ATHexapod', 'ATPneumatics', 'ATDomeTrajectory', 'ATAOS', 'ATMCS', 'ATDome'}.")
+
+The relevant information is highlighted in bold letters.
+In the example above, The ``ATPtg`` component timed out replying to the set authorization command from the ``Authorize`` CSC.
+In this case, this happened because authorization was not enabled in the component.
```

Comment:
I'm confused.
what does "authorization was not enabled" mean?

Reply:
uhm... that authorization is not enabled?

This feature can be switched on and off on the CSC. We are currently running with it disabled.

Reply:
Ah, so you can turn on/off the authorization in each component individually? If so, just say that.
Is there documentation that gives a high level overview of how the auth system works? If so it should be linked. If not, maybe just add a ~3 sentence summary at the top to give some more context?